media/live-seekable-range.js

```diff
@@ -0,0 +1,95 @@
+const video = document.querySelector('video');
+
+const mediaSource = new MediaSource();
+video.src = URL.createObjectURL(mediaSource);
+  
+mediaSource.addEventListener('sourceopen', function() {
+  URL.revokeObjectURL(video.src);
+
+  mediaSource.addSourceBuffer('video/webm; codecs="vp9"');
+
+  log('Fetching video init segment...');
+  fetchAndAppendSegment('bytes=0-299');
+});
```

Comment:
Make this handler only happen Once. Otherwise, error occurs later if endOfStream() followed by anything that re-opens the mediaSource (like append or remove) is done, because a re-opening sends a 'sourceopen' event again.

Example error:
Uncaught QuotaExceededError: Failed to execute 'addSourceBuffer' on 'MediaSource': This MediaSource has reached the limit of SourceBuffer objects it can handle. No additional SourceBuffer objects may be added. (Your browser may not support this feature.)